packages/iris-grid/src/IrisGridTableModel.js

```diff
@@ -1113,13 +1133,21 @@ class IrisGridTableModel extends IrisGridModel {
     const columnName = this.columns[x].name;
     if (
       this.layoutHints?.frontColumns?.includes(columnName) ||
-      this.layoutHints?.backColumns?.includes(columnName)
+      this.layoutHints?.backColumns?.includes(columnName) ||
+      this.layoutHints?.frozenColumns?.includes(columnName)
     ) {
       return false;
     }
     return !this.isKeyColumn(x);
   }
 
+  isColumnFrozen(x) {
+    if (this.layoutHints?.frozenColumns?.includes(this.columns[x].name)) {
+      return true;
+    }
+    return false;
```

Comment:
Can just return directly, no need for if:
```suggestion
    return this.layoutHints?.frozenColumns?.includes(this.columns[x].name) ?? false;
```